src/main/java/org/folio/consortia/controller/TenantController.java

```diff
@@ -78,11 +77,24 @@ public ResponseEntity<Void> createPrimaryAffiliations(UUID consortiumId, String
       SyncPrimaryAffiliationBody syncPrimaryAffiliationBody) {
     try {
       syncPrimaryAffiliationService.syncPrimaryUserAffiliations(consortiumId, centralTenantId, syncPrimaryAffiliationBody);
-      return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
+      return ResponseEntity.status(NO_CONTENT).build();
     } catch (Exception e) {
       log.error("createPrimaryAffiliations:: error creating user primary affiliations", e);
       tenantService.updateTenantSetupStatus(tenantId, centralTenantId, SetupStatusEnum.FAILED);
       throw e;
     }
   }
+
+  @Override
+  public ResponseEntity<Void> createIdentityProvider(UUID consortiumId, String tenantId) {
+    tenantManager.createIdentityProvider(tenantId);
```

Comment:
In next sprint we will add logic to migrate an existing users by invoking this endpoint
https://folio-org.atlassian.net/browse/MODUSERSKC-75

Related story: https://folio-org.atlassian.net/browse/MODCONSKC-70